OpenSim/Common/Reporter.h

```diff
@@ -163,10 +163,24 @@ OpenSim_DECLARE_CONCRETE_OBJECT_T(TableReporter_, InputT, Reporter<InputT>);
     TableReporter_() = default;
     virtual ~TableReporter_() = default;
 
+    /** Retrieve the report.                                                  */
```

Comment:
Did we rule out the "clear on read" case, @tkuchida. I thought that was handy in most cases and would avoid having to manually call clearReport(). Performing multiple simulations in a loop without calling `getReport()` seems like a case we need to cover (thus an explicit clearReport()) but I think calling `getReport()` will be the typical use case. Let me know if this was already discussed between you guys and you decided to not do that.